### ISSUE TITLE
context_processors arg to setup has wrong type annotation

### ISSUE THREAD

**Issue by bmerry** [aio-libs/aiohttp-jinja2#354]
Given this code, with aiohttp_jinja2 1.2.0:
```python
#!/usr/bin/env python3

import os

from aiohttp import web
import aiohttp_jinja2
import jinja2

@aiohttp_jinja2.template('foo.txt')
def handler(request):
    return {}

app = web.Application()
aiohttp_jinja2.setup(
    app,
    loader=jinja2.FileSystemLoader(os.path.abspath('./templates')),
    context_processors=[aiohttp_jinja2.request_processor]
)
app.router.add_get('/foo.txt', handler)
web.run_app(app)
```

mypy reports this error:
```
demo.py:17: error: List item 0 has incompatible type "Callable[[Request], Coroutine[Any, Any, Dict[str, Request]]]"; expected "Callable[[Request], Dict[str, Any]]"
```

**Comment by Dreamsorcerer** [comment]
I don't have the code setup right now to test it, but I suspect this line just needs to be updated to expect an awaitable:
https://github.com/aio-libs/aiohttp-jinja2/blob/master/aiohttp_jinja2/__init__.py#L51

`context_processors: Iterable[Callable[[web.Request], Awaitable[Dict[str, Any]]]] = ()`

**Comment by asvetlov** [comment]
Please feel free to prepare a pull request

**Comment by Dreamsorcerer** [comment]
Tested out the fix, seems to be correct.

**Comment by asvetlov** [comment]
Fixed by #386
